index.js

```diff
@@ -102,11 +103,16 @@ const init = (seed, option) => {
                 }));
             }));
         }));
-
         r.net.Connection.prototype.DEFAULT_PORT = port;
         t.context.dbPort = port;
     };
 };
+const restore = async (option) => {
+    const filepath = await tempWrite(option.password);
+    console.log('Restoring data from skeleton.tar.gz');
+    await execa('rethinkdb', ['restore', option.dump, '--password-file', filepath, '--connect', `localhost:${option.port}`]);
```

Comment:
Use `execa`'s `preferLocal : false` option, like the other place where we run a RethinkDB command.

Reply:
Ok